src/gwt/src/org/rstudio/studio/client/workbench/ui/WorkbenchTabPanel.java

```diff
@@ -44,11 +48,29 @@
 {
    public WorkbenchTabPanel(WindowFrame owner, LogicalWindow parentWindow)
    {
+      final int UTILITY_AREA_SIZE = 52;
+      panel_ = new LayoutPanel();
+      
       parentWindow_ = parentWindow;
+
       tabPanel_ = new ModuleTabLayoutPanel(owner);
+      // tabPanel_ = new DocTabLayoutPanel(true, 65, UTILITY_AREA_SIZE);
```

Comment:
Do you think we'll need this code in the future? (can we remove it if not, or add an auxiliary comment if so?)

Reply:
Oops, no that should be removed, fixed this one while my wife was driving us to Portland =P Removed.

Reply:
Glad to hear it wasn't while *you* were driving. Safety first!